### ISSUE TITLE
Vulnerable Dependency: commons-beanutils 1.9.3 is affected by CVE-2019-10086

### ISSUE THREAD

**Issue by gtudan** [eclipse-ee4j/krazo#92]
The dependency check fails because of a vulnerable dependency:

```
[ERROR] Failed to execute goal org.owasp:dependency-check-maven:5.1.0:aggregate (default-cli) on project krazo-parent: 
[ERROR] 
[ERROR] One or more dependencies were identified with vulnerabilities that have a CVSS score greater than or equal to '7.0': 
[ERROR] 
[ERROR] commons-beanutils-1.9.3.jar: CVE-2019-10086
[ERROR] 
[ERROR] See the dependency-check report for more details.
```

The dependency is introduced by the velocity-extension:

```
[INFO] --- maven-dependency-plugin:2.8:tree (default-cli) @ krazo-velocity ---
[INFO] org.eclipse.krazo.ext:krazo-velocity:jar:1.0.0-SNAPSHOT
[INFO] +- org.apache.velocity.tools:velocity-tools-view:jar:3.0:compile
[INFO] |  +- org.apache.velocity:velocity-engine-core:jar:2.0:compile
[INFO] |  +- org.apache.velocity.tools:velocity-tools-generic:jar:3.0:compile
[INFO] |  |  +- commons-beanutils:commons-beanutils:jar:1.9.3:compile
```

A fix has been provided in BeanUtils 1.9.4 and should be binary compatible with any previous 1.9.x release: http://commons.apache.org/proper/commons-beanutils/javadocs/v1.9.4/RELEASE-NOTES.txt

**Comment by chkal** [comment]
Maybe we should just add a `dependencyManagent` entry to update the version for beanutils?

**Comment by erdlet** [comment]
`dependencyManagement` seems to work. Good idea :)

**Comment by gtudan** [comment]
I'm okay with the dependency-management fix. 
But maybe we can port the extension from velocity-tools to pure velocity-engine some time in the future to get rid of those shady dependencies? While the engine is pretty slim on dependencies, the tools bring in quiet a lot of stuff that we don't really need. In fact its just a single class (the resource loader).

It's been some time since the latest velocity-tools release and there are newer engine releases available.

**Comment by chkal** [comment]
@gtudan I agree. We already had issues with velocity-tools in the past, because they didn't provide a release for Velocity 2.0 in time. So maybe getting rid of it would be the best option. Could you create an issue to track this?

**Comment by chkal** [comment]
Fixed by #93
